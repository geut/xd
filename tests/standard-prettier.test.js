const runCommonTests = require('.')

runCommonTests('standard-prettier', '--prettier')
