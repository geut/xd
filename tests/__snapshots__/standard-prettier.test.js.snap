// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`standard-prettier: fix file fix --fix -f json 1`] = `"[{\\"filePath\\":\\"/tmp/xd-standard-prettier-test/index.js\\",\\"messages\\":[],\\"errorCount\\":0,\\"warningCount\\":0,\\"fixableErrorCount\\":0,\\"fixableWarningCount\\":0,\\"output\\":\\"function HelloWorld({\\\\n  greeting = 'hello',\\\\n  greeted = '\\\\\\"World\\\\\\"',\\\\n  silent = false,\\\\n  onMouseOver\\\\n}) {\\\\n  if (!greeting) {\\\\n    return null\\\\n  }\\\\n}\\\\n\\\\nHelloWorld()\\\\n\\"}]"`;

exports[`standard-prettier: fix file fix --fix-dry-run -f json 1`] = `"[{\\"filePath\\":\\"/tmp/xd-standard-prettier-test/index.js\\",\\"messages\\":[],\\"errorCount\\":0,\\"warningCount\\":0,\\"fixableErrorCount\\":0,\\"fixableWarningCount\\":0,\\"output\\":\\"function HelloWorld({\\\\n  greeting = 'hello',\\\\n  greeted = '\\\\\\"World\\\\\\"',\\\\n  silent = false,\\\\n  onMouseOver\\\\n}) {\\\\n  if (!greeting) {\\\\n    return null\\\\n  }\\\\n}\\\\n\\\\nHelloWorld()\\\\n\\"}]"`;

exports[`standard-prettier: fix file fix --stdin --fix-dry-run -f json 1`] = `"[{\\"filePath\\":\\"<text>\\",\\"messages\\":[],\\"errorCount\\":0,\\"warningCount\\":0,\\"fixableErrorCount\\":0,\\"fixableWarningCount\\":0,\\"output\\":\\"function HelloWorld({\\\\n  greeting = 'hello',\\\\n  greeted = '\\\\\\"World\\\\\\"',\\\\n  silent = false,\\\\n  onMouseOver\\\\n}) {\\\\n  if (!greeting) {\\\\n    return null\\\\n  }\\\\n}\\\\n\\\\nHelloWorld()\\\\n\\"}]"`;

exports[`standard-prettier: fix file fix --stdin --stdin-filename "filename" --fix-dry-run -f json 1`] = `"[{\\"filePath\\":\\"/tmp/xd-standard-prettier-test/index.js\\",\\"messages\\":[],\\"errorCount\\":0,\\"warningCount\\":0,\\"fixableErrorCount\\":0,\\"fixableWarningCount\\":0,\\"output\\":\\"function HelloWorld({\\\\n  greeting = 'hello',\\\\n  greeted = '\\\\\\"World\\\\\\"',\\\\n  silent = false,\\\\n  onMouseOver\\\\n}) {\\\\n  if (!greeting) {\\\\n    return null\\\\n  }\\\\n}\\\\n\\\\nHelloWorld()\\\\n\\"}]"`;

exports[`standard-prettier: lint file lint --stdin -f json 1`] = `
"[{\\"filePath\\":\\"/tmp/xd-standard-prettier-test/index.js\\",\\"messages\\":[{\\"ruleId\\":\\"prettier/prettier\\",\\"severity\\":2,\\"message\\":\\"Replace \`·({·greeting·=·'hello',·greeted·=·'\\\\\\"World\\\\\\"',·silent·=·false,·onMouseOver·\` with \`({⏎··greeting·=·'hello',⏎··greeted·=·'\\\\\\"World\\\\\\"',⏎··silent·=·false,⏎··onMouseOver⏎\`\\",\\"line\\":1,\\"column\\":20,\\"nodeType\\":null,\\"endLine\\":1,\\"endColumn\\":93,\\"fix\\":{\\"range\\":[19,92],\\"text\\":\\"({\\\\n  greeting = 'hello',\\\\n  greeted = '\\\\\\"World\\\\\\"',\\\\n  silent = false,\\\\n  onMouseOver\\\\n\\"}},{\\"ruleId\\":\\"prettier/prettier\\",\\"severity\\":2,\\"message\\":\\"Replace \`·return·null;\` with \`⏎····return·null⏎·\`\\",\\"line\\":2,\\"column\\":19,\\"nodeType\\":null,\\"endLine\\":2,\\"endColumn\\":32,\\"fix\\":{\\"range\\":[115,128],\\"text\\":\\"\\\\n    return null\\\\n \\"}},{\\"ruleId\\":\\"prettier/prettier\\",\\"severity\\":2,\\"message\\":\\"Delete \`;\`\\",\\"line\\":3,\\"column\\":2,\\"nodeType\\":null,\\"endLine\\":3,\\"endColumn\\":3,\\"fix\\":{\\"range\\":[132,133],\\"text\\":\\"\\"}},{\\"ruleId\\":\\"prettier/prettier\\",\\"severity\\":2,\\"message\\":\\"Delete \`;\`\\",\\"line\\":5,\\"column\\":13,\\"nodeType\\":null,\\"endLine\\":5,\\"endColumn\\":14,\\"fix\\":{\\"range\\":[147,148],\\"text\\":\\"\\"}}],\\"errorCount\\":4,\\"warningCount\\":0,\\"fixableErrorCount\\":4,\\"fixableWarningCount\\":0,\\"source\\":\\"function HelloWorld ({ greeting = 'hello', greeted = '\\\\\\"World\\\\\\"', silent = false, onMouseOver }) {\\\\n  if (!greeting) { return null; }\\\\n};\\\\n\\\\nHelloWorld();\\\\n\\"}]
"
`;

exports[`standard-prettier: lint file lint -f json 1`] = `
"[{\\"filePath\\":\\"/tmp/xd-standard-prettier-test/index.js\\",\\"messages\\":[{\\"ruleId\\":\\"prettier/prettier\\",\\"severity\\":2,\\"message\\":\\"Replace \`·({·greeting·=·'hello',·greeted·=·'\\\\\\"World\\\\\\"',·silent·=·false,·onMouseOver·\` with \`({⏎··greeting·=·'hello',⏎··greeted·=·'\\\\\\"World\\\\\\"',⏎··silent·=·false,⏎··onMouseOver⏎\`\\",\\"line\\":1,\\"column\\":20,\\"nodeType\\":null,\\"endLine\\":1,\\"endColumn\\":93,\\"fix\\":{\\"range\\":[19,92],\\"text\\":\\"({\\\\n  greeting = 'hello',\\\\n  greeted = '\\\\\\"World\\\\\\"',\\\\n  silent = false,\\\\n  onMouseOver\\\\n\\"}},{\\"ruleId\\":\\"prettier/prettier\\",\\"severity\\":2,\\"message\\":\\"Replace \`·return·null;\` with \`⏎····return·null⏎·\`\\",\\"line\\":2,\\"column\\":19,\\"nodeType\\":null,\\"endLine\\":2,\\"endColumn\\":32,\\"fix\\":{\\"range\\":[115,128],\\"text\\":\\"\\\\n    return null\\\\n \\"}},{\\"ruleId\\":\\"prettier/prettier\\",\\"severity\\":2,\\"message\\":\\"Delete \`;\`\\",\\"line\\":3,\\"column\\":2,\\"nodeType\\":null,\\"endLine\\":3,\\"endColumn\\":3,\\"fix\\":{\\"range\\":[132,133],\\"text\\":\\"\\"}},{\\"ruleId\\":\\"prettier/prettier\\",\\"severity\\":2,\\"message\\":\\"Delete \`;\`\\",\\"line\\":5,\\"column\\":13,\\"nodeType\\":null,\\"endLine\\":5,\\"endColumn\\":14,\\"fix\\":{\\"range\\":[147,148],\\"text\\":\\"\\"}}],\\"errorCount\\":4,\\"warningCount\\":0,\\"fixableErrorCount\\":4,\\"fixableWarningCount\\":0,\\"source\\":\\"function HelloWorld ({ greeting = 'hello', greeted = '\\\\\\"World\\\\\\"', silent = false, onMouseOver }) {\\\\n  if (!greeting) { return null; }\\\\n};\\\\n\\\\nHelloWorld();\\\\n\\"}]
"
`;
